lib/Plack/Middleware/EnsThrottle.pm

```diff
@@ -45,6 +45,8 @@ enable 'EnsThrottle::Second',
   },
   blacklist => '192.168.2.1', #blacklist a single IP
   whitelist => ['192.168.0.0-192.169.255.255'], #whitelist a range
+  whitelist_hdr => 'Token',
```

Comment:
That's an example and in the documentation section, the "live" config is of course hidden in the private github. I could change it to something more accurate, at the same time I'm not sure I want anything close to what the "real" value we use will be :)  "X-My-Secret-Header"?